### PR TITLE
fix: [CDS-69175]: HF for putting HTTP url validation behind FF

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger standards

**Comment by satyam-harness** [comment]
retrigger build

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -279,4 +279,4 @@
   "jest-junit": {
     "addFileAttribute": "true"
   }
-}
+}
```
Comment (AshwinBhatkal-Harness) [inline package.json:282]: Add this extra line

Comment (AshwinBhatkal-Harness) [inline package.json:282]: The extra line has been removed here? 🤔

Comment (satyam-harness) [reply package.json:282]: due to prettier, it got auto-removed. Shall i add it?

**Review by litvinovstas — APPROVED** [review]

**Pull request merged**